src/oplconstants.h

```diff
@@ -0,0 +1,113 @@
+#ifndef OPLCONSTANTS_H
+#define OPLCONSTANTS_H
+
+#include <QtCore>
+
+/*!
+ *  A namespace to collect constants used throughout the application.
+ */
+namespace opl {
+
+/*!
+ * The datetime namespace specifies date and time formats by QDate, QTime
+ * and QDateTime that deviate from standard values included in the Qt Framework
+ * like Qt::ISODate and are to be used in conjunction with the .toString() members
+ * of these classes
+ */
+namespace datetime {
+
+static const auto TIME_DEFAULT_FORMAT    = QStringLiteral("hh:mm");
```

Comment:
Can just use Qt::TextDate for uniformity.
https://doc.qt.io/qt-5/qtime.html#toString-1

Reply:
Qt::ISODate returns HH:mm:ss, Flight Times are always logged only to minutes in accuracy. Since we are not logging, using or displaying seconds, Qt::ISODate is unsuitable.

Reply:
Then since this is a domain specific thing it should be named as such.
FLIGHT_TIME_FORMAT seems more suitable

Reply:
Hm then we would another one for the general default. It also applies to departure time, arrival time etc. But having FLIGHT_TIME_DEFAULT_FORMAT and TIME_DEFAULT_FORMAT is not a bad idea, since we may include[additional flight time formats](https://github.com/fiffty-50/openpilotlog/wiki/Future-Features-ideas#3-support-for-decimal-time-support) in the future. I'll change it.